modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -301,6 +301,10 @@
       "XDB"
       "XS$NULL"}))
 
+(defmethod driver/metadata-escape-entity-name :oracle
+  [_driver table-name]
```

Comment:
We usually just use `_`. This is super minor, but when I first read for a moment I though that the fn takes 3 args.

Also perhaps `table-or-schema` or `entity-name` would be names that better match how the fn is used.